Source/WebKit/Shared/RemoteLayerTree/LayerProperties.h

```diff
@@ -77,14 +77,14 @@ enum class LayerChange : uint64_t {
     ScrollingNodeIDChanged              = 1LLU << 39,
 #endif
 #if HAVE(CORE_ANIMATION_SEPARATED_LAYERS)
-    SeparatedChanged                    = 1LLU << 39,
+    SeparatedChanged                    = 1LLU << 40,
```

Comment:
Should we change this because afaik HAVE_CORE_ANIMATION_SEPARATED_LAYERS is not 1 when ENABLE_SCROLLING_THREAD is 1 but we had not verification of that previously?

Reply:
Will that always be the case? Maybe it should be `#if #else` then?